ideascube/tests/helpers.py

```diff
@@ -0,0 +1,37 @@
+from django.apps import apps
+from django.test import TransactionTestCase
+from django.db.migrations.executor import MigrationExecutor
+from django.db import connection
+import pytest
+
+
+@pytest.mark.skipif(
+    pytest.config.getoption("--nomigrations"),
+    reason="skipping due to --nomigrations"
+)
+class MigrationTestCase(TransactionTestCase):
```

Comment:
I'd have used pytest fixture instead of going for unittest classes here, see http://docs.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code for managing setup/teardown (more readable and maintainable *imho*, and certainly more consistent with the way we usually do in other tests).

Is there a reason why you didn't go that way?